public/app/features/dashboard-scene/saving/SaveDashboardAsForm.tsx

```diff
@@ -85,7 +85,16 @@ export function SaveDashboardAsForm({ dashboard, changeInfo }: Props) {
 
   const saveButton = (overwrite: boolean) => {
     const showSaveButton = !isValid && hasFolderChanged ? true : isValid;
-    return <SaveButton isValid={showSaveButton} isLoading={state.loading} onSave={onSave} overwrite={overwrite} />;
+    const isTitleValidating = !!validatingFields.title;
+
+    return (
+      <SaveButton
```

Comment:
We should add a property to customize the button content or at least the loading text. It is weird to have "Saving..." when you open the modal.


```suggestion
      <SaveButton
        loadingText={"foo"}
```